### DISCUSSION TITLE
v1.6.3

### DISCUSSION THREAD

**Discussion by LazyImmortal** [LazyImmortal/Sesame#10] [Announcements]
## 更新说明

添加 本地模式支持(Taichi and LSPatch)

## 下载说明

- Normal 为正常版本，适用于**Android 8.0**及以上的系统
- Compatible 为兼容版本，适用于**Android 7.0**及以下的系统，最低支持**Android 5.1**

<hr /><em>This discussion was created from the release <a href='https://github.com/LazyImmortal/Sesame/releases/tag/v1.6.3'>v1.6.3</a>.</em>

**Comment by Andyqiu-rm** [comment]
大佬回来了👍

**Comment by Abysslyz** [comment]
打开支付宝显示芝麻关门：四十大盗，软件显示已关闭。这是要给什么权限或者改什么设置吗

**Reply by xiaohuihui-ht** [reply]
你没root吧

**Reply by Zhangfanyu1222** [reply]
你是内置版吧？

**Reply by Abysslyz** [reply]
> 你没root吧

这个手机需要root才能用，这不是不用root的情况下可用的吗？

**Reply by Abysslyz** [reply]
> 你是内置版吧？

内置和本地都试了，一样的结果。就是不知道是哪里的问题，用TK版加载成功了，用大佬的就是提示芝麻关门：四十大盗加载不了。

**Reply by JaredHHu** [reply]
使用Lspatch集成模式的时候会显示“芝麻关门：四十大盗”，但换成本地模式就能正常运行。
环境：Origin OS5，没有root，支v10.6.70.8100。

**Reply by Abysslyz** [reply]
> 使用Lspatch集成模式的时候会显示“芝麻关门：四十大盗”，但换成本地模式就能正常运行。 环境：Origin OS5，没有root，支v10.6.70.8100。

集成和本地都试过了，一样的情况。不知道是什么原因

**Reply by tangjie602** [reply]
> 你没root吧

要root才能用是吗

**Reply by Zhangfanyu1222** [reply]
> > 你没root吧
> 
> 要root才能用是吗

不root用本地模式

**Reply by tangjie602** [reply]
> > > 你没root吧
> > 
> > 
> > 要root才能用是吗
> 
> 不root用本地模式

lsp本地模式怎么操作
![Uploading Screenshot_20241224_163137.jpg…]()

**Reply by Abysslyz** [reply]
> > > 你没root吧
> > 
> > 
> > 要root才能用是吗
> 
> 不root用本地模式

本地模式试过了还是一样，1.6.4之后本地模式多了一个不可用的安装包。难道就只能Root后才能用了吗

**Reply by RolayXL** [reply]
> > 你是内置版吧？
> 
> 内置和本地都试了，一样的结果。就是不知道是哪里的问题，用TK版加载成功了，用大佬的就是提示芝麻关门：四十大盗加载不了。

我跟你一样的问题，zfb各个版本全试过了都不行，已经用回TK版了😂

**Comment by Tsn-09** [comment]
回来啦！

**Comment by tangjie602** [comment]
现在这个要怎么使用呢

**Comment by mem4tgo** [comment]
up这个不能后台领视频红包吗

**Reply by Zhangfanyu1222** [reply]
> up这个不能后台领视频红包吗

这边不做薅羊毛类的😂

**Reply by berebrn** [reply]
> up这个不能后台领视频红包吗

你是不是脑残

**Reply by mem4tgo** [reply]
> > up这个不能后台领视频红包吗
> 
> 你是不是脑残

怎么了？问问不可以？芝麻X就可以刷。每两周一个60元红包。我只是好奇问问咋刺激你了？

**Reply by berebrn** [reply]
> > > up这个不能后台领视频红包吗
> > 
> > 
> > 你是不是脑残
> 
> 怎么了？问问不可以？芝麻X就可以刷。每两周一个60元红包。我只是好奇问问咋刺激你了？

你就是给傻逼，开源说明很明确，别人都知道就你不知道，你是不是傻逼啊，要是你喜欢薅羊毛可以自己fork

**Reply by berebrn** [reply]
> > > up这个不能后台领视频红包吗
> > 
> > 
> > 你是不是脑残
> 
> 怎么了？问问不可以？芝麻X就可以刷。每两周一个60元红包。我只是好奇问问咋刺激你了？

自己没有那个实力就要求大佬去增加这些那些的，大佬欠你的吗？爱用不用，还他妈的给你狗脸了啊

**Reply by mem4tgo** [reply]
> > > > up这个不能后台领视频红包吗
> > > 
> > > 
> > > 你是不是脑残
> > 
> > 
> > 怎么了？问问不可以？芝麻X就可以刷。每两周一个60元红包。我只是好奇问问咋刺激你了？
> 
> 自己没有那个实力就要求大佬去增加这些那些的，大佬欠你的吗？爱用不用，还他妈的给你狗脸了啊

杂种玩意。一看就是东北日本杂角碎

**Reply by mem4tgo** [reply]
> > > > up这个不能后台领视频红包吗
> > > 
> > > 
> > > 你是不是脑残
> > 
> > 
> > 怎么了？问问不可以？芝麻X就可以刷。每两周一个60元红包。我只是好奇问问咋刺激你了？
> 
> 自己没有那个实力就要求大佬去增加这些那些的，大佬欠你的吗？爱用不用，还他妈的给你狗脸了啊

真不知道你算个JB。找你惹你了？UP也不是你爹，你想认up当爹他都觉得你是东北日本血的杂种

**Reply by berebrn** [reply]
> > > > > up这个不能后台领视频红包吗
> > > > 
> > > > 
> > > > 你是不是脑残
> > > 
> > > 
> > > 怎么了？问问不可以？芝麻X就可以刷。每两周一个60元红包。我只是好奇问问咋刺激你了？
> > 
> > 
> > 自己没有那个实力就要求大佬去增加这些那些的，大佬欠你的吗？爱用不用，还他妈的给你狗脸了啊
> 
> 真不知道你算个JB。找你惹你了？UP也不是你爹，你想认up当爹他都觉得你是东北日本血的杂种

我又不是你爹，我骂的就是你这个日本血的杂种，纯傻逼玩儿
